wgengine/router/router_openbsd.go

```diff
@@ -228,6 +228,10 @@ func (r *openbsdRouter) Set(cfg *Config) error {
 	return errq
 }
 
+func (r *openbsdRouter) UpdateMagicsockPort(_ uint16, _ string) error {
+	return nil
+}
```

Comment:
For these, I would leave a comment why this is returning nil. I believe we have done that in a few other places to make it explicit that this isn't needed on this platform/etc. It just helps distinguish between it being intentional and 'whoops I was meant to get back to this'.